src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface DrawFigure {
```

Comment:
No need to tight interface name for figure only
```suggestion
interface Drawable {
```